### ISSUE TITLE
Re-write the whole project as it is based on JRuby repo

### ISSUE THREAD

**Issue by jmardjuki** [TheChyz/Mini-Basketball-League#1]
Does not run on Bobby's VM.
We might be able to try docker, and use  Heroku-JRuby docker image  but is that what we want?

**Comment by MichaelChyziak** [comment]
Since all we have is the template we can just copy the normal heroku setting started project and continue from there. All we really changed is the README. I think this is easier than trying to use the heroku jruby docker image

**Comment by jmardjuki** [comment]
I will work on this most likely by tomorrow.
Unless someone else wants to work on it.

**Comment by MichaelChyziak** [comment]
You can go ahead. Its basically just clone the entire getting_started project (but not the jruby one i did) and just copy and paste the README

**Comment by MichaelChyziak** [comment]
Where did you clone this project from? It looks a little different then the one that bobby links to which is https://github.com/heroku/ruby-getting-started.git. And what is the docker for?

**Comment by jmardjuki** [comment]
I replaced all the stuff with the stuff in that github repo.
The only extra thing I added is Docker stuff. Docker is used to run the app locally in a virtual container.
As I told you guys, I'm still having problem running locally using Bobby's VM. It does run, but stuck at the database stuff with no privilege. I have configured my own VM before the second assignment with a lot of trial and error; I can check the history but getting it to work would take too much time. We need to assign someone to try configuring the db on Bobby's VM, else I would suggest everyone to use Docker.

**Comment by MichaelChyziak** [comment]
I can try to configure it using Bobby's VM since that will keep everything consistent between all of our environments. However I won't get to this until tmrw. I'll keep you guys posted if I get anything working or if not.

**Comment by jmardjuki** [comment]
I was thinking of closing this issue and open a new issue issue, but I guess db issue is still very much related to this issue. 
Please close this issue if you are able to run it on your machine without the db problems.
Docker does keep everything consistent by the way.

**Comment by MichaelChyziak** [comment]
I got the VM to run the webapp locally and forced the push to the master. If there are any problems just message me on facebook since I'm probably the most experienced with getting the current version working. Make sure to pull the latest header before starting your work.

**Comment by jmardjuki** [comment]
Follow Bobby's guide to resolve the db issue
http://www.cs.sfu.ca/CourseCentral/276/bobbyc/n/tutorials/database-for-local-development.pdf
We are using the the default name for the db, use the following to create the db

```
$ sudo -u postgres psql
# create database ruby_getting_started_development;
# create database ruby_getting_started_test;
# create database ruby_getting_started;
```
